pkg/orchestrator/etcd_worker.go

```diff
@@ -229,7 +230,9 @@ func (worker *EtcdWorker) Run(ctx context.Context, session *concurrency.Session,
 				// handleEvent will apply the event to our internal `rawState`.
 				err = worker.handleEvent(ctx, event)
 				if err != nil {
-					return err
+					// This error means owner is resigned by itself,
+					// and we should exit etcd worker and campaign owner again.
```

Comment:
log the error

Reply:
This error is log in the `handleEvent` func.